README.md

```diff
@@ -64,6 +64,22 @@ Install the version of node.js supported by Studio, and install `yarn` version 1
 volta install node@16
 volta install yarn@1
 ```
+Alternatively,
+```bash
+# Installing node@16 and yarn@1 using node-env
+pip install nodeenv
+nodeenv -l
+function latest-node() { curl -s "https://nodejs.org/dist/latest-v$1.x/" | egrep -m 1 -o "$1\.[0-9]+\.[0-9]+" | head -1; }
+latest-node 16
+nodeenv --python-virtualenv --node 16.16.0
```

Comment:
Assuming that we are using "--python-virtualenv" argument to bind the nodeenv with the python env, it would be helpful to tell the user to activate the python environment before using this command?